bridge_adaptivity/requirements.txt

```diff
@@ -1,5 +1,6 @@
 # install base requirements
 -r requirements_base.txt
+-r requirements_local.txt  # FIXME(wowkalucky): should add it here to satisfy debug_toolbar module import (somewhere)
```

Comment:
This is Strange Thing...

Reply:
Please remove included requirements_local from the requirements file it looks confusing.

Reply:
Please remove included requirements_local from the requirements file it looks confusing.